.travis.yml

```diff
@@ -1,8 +1,5 @@
 language: go
 
-go:
-  - 1.4
```

Comment:
The latest stable release of Go in travis is 1.4.x (see the build results for this pull request).